cbor2/compat.py

```diff
@@ -47,3 +49,60 @@ def iteritems(self):
     long = int  # noqa
     unicode = str  # noqa
     bytes_from_list = bytes
+
+
+def pack_float16(value):
```

Comment:
Why not have the test on the module level? The test suite is run on 3.6 as well as older Pythons so both code paths should be covered in the end.

Reply:
will do, I've never used test coverage tools before as you can probably tell!